examples/lighting-app/mbed/README.md

```diff
@@ -0,0 +1,235 @@
+## Testing the example
+
+Check the [**LINK HERE**] to learn how to use command-line interface 
+of the application.
+
+<a name="testing_chiptool"></a>
+## Testing using CHIPTool
+
+Read the
+[**LINK HERE**] to see how to use 
+[CHIPTool](../../../src/android/CHIPTool/README.md) for
+Android smartphones to commission and control the application 
+within a Wi-Fi network.
+
+<a name="supported_devices"></a>
+## Supported devices
+
+The example supports building and running on the following mbed-enabled devices:
+
+| Manufacturer | Hardware platform | Build target | Platform image | Status |
+| ------------ | ----------------- | ------------ | -------------- | :----: |
+| [Cypress<br> Semiconductor](https://www.cypress.com/) | [CY8CPROTO-062-4343W](https://os.mbed.com/platforms/CY8CPROTO-062-4343W/) | `CY8CPROTO_062_4343W` | <details><summary>CY8CPROTO-062-4343W</summary><img src="https://os.mbed.com/media/cache/platforms/p6_wifi-bt_proto.png.250x250_q85.jpg" alt="CY8CPROTO-062-4343W"/></details> | :heavy_check_mark: |
+
+
+There is a plan to expend above list with new platforms within mbed-os ecosystem.
+
+<a name="notes"></a>
+#### Notes
+
+* More details and guidelines about porting new hardware into the CHIP project with mbed-os can be found in **[LINK HERE]**
+* Some useful information about HW platform specific settings can be found in `lighting-app/mbed/mbed_app.json`.    
+Information about this file syntax and its meaning in mbed-os project can be found here: [Mbed-Os configuration system](https://os.mbed.com/docs/mbed-os/v6.12/program-setup/advanced-configuration.html))
+
+
+ <hr>
+<a name="device_ui"></a>
+# Device UI
+
+This section lists the User Interface elements that you can use to control and
+monitor the state of the device. These correspond to PCB components on the
+platform image.
+
+** USER LED ** shows the overall state of the device and its connectivity. The
+following states are possible:
+
+-   _Short Flash On (50 ms on/950 ms off)_ &mdash; The device is in the
+    unprovisioned (unpaired) state and is waiting for a commissioning
+    application to connect.
+
+-   _Rapid Even Flashing (100 ms on/100 ms off)_ &mdash; The device is in the
+    unprovisioned state and a commissioning application is connected through
+    Bluetooth LE.
+
+-   _Short Flash Off (950ms on/50ms off)_ &mdash; The device is fully
+    provisioned, but does not yet have full network or service
+    connectivity.
+
+-   _Solid On_ &mdash; The device is fully provisioned and has full
+    network and service connectivity.
+
+**Lighting LED** simulates the light bulb and shows the state of the lighting. The
+following states are possible:
+
+-   _Solid On_ &mdash; The light bulb is on.
+
+-   _Off_ &mdash; The light bulb is off.
+
+-   _PWM_ &mdash; The light bulb is dimmed according to PWM.
+
+This led is not present on the target board but it can be connected to P9_6 pin.
+
+** Capacitive Button 0** can be used for the following purposes:
```

Comment:
This bold text is not rendered properly.

Reply:
done